| Arguments Keep work_dir: C:\Users\JK\PycharmProjects\2025_AAAI\work_dir/dj30_AAAI_AAAI_adam_mse\2025_AAAI_Exp_None_exp_num_0726/130316
rank:None start_date:['2012-02-02']
Epoch 1, train_loss 1.247002, valid_loss 0
Validation loss decreased (inf --> 1.273224).  Saving model ...
rank:None start_date:['2012-02-02']
Epoch 2, train_loss 1.278878, valid_loss 0
Validation loss decreased (1.273224 --> 1.270522).  Saving model ...
rank:None start_date:['2012-02-02']
Epoch 3, train_loss 1.241909, valid_loss 0
Validation loss decreased (1.270522 --> 1.267911).  Saving model ...
rank:None start_date:['2012-02-02']
Epoch 4, train_loss 1.243511, valid_loss 0
Validation loss decreased (1.267911 --> 1.265382).  Saving model ...
rank:None start_date:['2012-02-02']
Epoch 5, train_loss 1.247978, valid_loss 0
Validation loss decreased (1.265382 --> 1.262891).  Saving model ...
rank:None start_date:['2012-02-02']
Epoch 6, train_loss 1.210312, valid_loss 0
Validation loss decreased (1.262891 --> 1.260457).  Saving model ...
rank:None start_date:['2012-02-02']
Epoch 7, train_loss 1.204979, valid_loss 0
Validation loss decreased (1.260457 --> 1.258088).  Saving model ...
rank:None start_date:['2012-02-02']
Epoch 8, train_loss 1.211296, valid_loss 0
Validation loss decreased (1.258088 --> 1.255766).  Saving model ...
rank:None start_date:['2012-02-02']
Epoch 9, train_loss 1.182395, valid_loss 0
Validation loss decreased (1.255766 --> 1.253551).  Saving model ...
rank:None start_date:['2012-02-02']
Epoch 10, train_loss 1.188797, valid_loss 0
Validation loss decreased (1.253551 --> 1.251391).  Saving model ...
Successfully loaded best checkpoint...
Test Best Episode
Traceback (most recent call last):
  File "C:\Program Files\JetBrains\PyCharm 2021.1.3\plugins\python\helpers\pydev\pydevd.py", line 1551, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "C:\Program Files\JetBrains\PyCharm 2021.1.3\plugins\python\helpers\pydev\_pydev_imps\_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "C:\Users\JK\PycharmProjects\2025_AAAI\run.py", line 239, in <module>
    run(cfg)
  File "C:\Users\JK\PycharmProjects\2025_AAAI\run.py", line 186, in run
    trainer.test()
  File "C:\Users\JK\PycharmProjects\2025_AAAI\trainers\AAAI_mse.py", line 305, in test
    gathered_preds = torch.cat(gathered_preds).cpu().numpy()
TypeError: cat(): argument 'tensors' (position 1) must be tuple of Tensors, not Tensor